### PR TITLE
zh-cn: translation for webusb api

### PR DESCRIPTION
**Pull request by xuanzhi33**
### Description

translation for webusb api

### CODE REVIEW

File: files/zh-cn/web/api/webusb_api/index.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+});
+```
+
+## 规范
+
+{{Specifications}}
+
+## 参见
+
+- [Access USB Devices on the Web](https://developer.chrome.com/docs/capabilities/usb)
```
Comment (T34-active) [inline files/zh-cn/web/api/webusb_api/index.md:88]:
```suggestion
- [通过网页访问 USB 设备](https://developer.chrome.com/docs/capabilities/usb)
```

File: files/zh-cn/web/api/webusb_api/index.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+  sourceCommit: 4458494807b6f4898d504b6c0af0a45f8031cbf3
+---
+
+{{securecontext_header}}{{DefaultAPISidebar("WebUSB API")}}{{SeeCompatTable}}
+
+**WebUSB API** 提供了一种将非标准通用串行总线（USB）兼容设备服务暴露到 web 的方法，使 USB 更安全，更易于使用。
+
+## 概念和用法
+
+USB 是有线外设的事实上的标准。连接到计算机的 USB 设备通常分为许多类型：如键盘、鼠标、视频设备等。这些是使用操作系统的类驱动程序支持的。其中许多也可以在 web 中通过 {{domxref("WebHID API")}} 进行访问。
```
Comment (yin1999) [inline files/zh-cn/web/api/webusb_api/index.md:14]:
```suggestion
USB 是有线外设的事实标准。连接到计算机的 USB 设备通常分为许多类型：如键盘、鼠标、视频设备等。这些是使用操作系统的类驱动程序支持的。其中许多也可以在 web 中通过 {{domxref("WebHID API")}} 进行访问。
```

File: files/zh-cn/web/api/webusb_api/index.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+除了这些标准化的设备之外，还有大量的设备不适合任何类别。这些设备需要自定义驱动程序，并且由于需要本机代码，因此无法从 web 访问。安装这些设备，通常需要在制造商的网站上搜索驱动程序，如果你想在另一台电脑上使用该设备，还必须重复这个过程。
```
Comment (yin1999) [inline files/zh-cn/web/api/webusb_api/index.md:16]:
```suggestion
除了这些标准化的设备之外，还有大量的设备不属于任何类别。这些设备需要自定义驱动程序，并且由于需要机器代码（native code），因此无法从 web 访问。安装这些设备，通常需要在制造商的网站上搜索驱动程序，如果你想在另一台电脑上使用该设备，还必须重复这个过程。
```

**Review by yin1999 — COMMENTED** [review]
nits:

**Review by yin1999 — APPROVED** [review]
Thanks :+1:

**Pull request merged**